admin/includes/languages/lang.english.php

```diff
@@ -9,7 +9,7 @@
 @setlocale(LC_TIME, ['en_US', 'en_US.utf8', 'en', 'English_United States.1252']);
 
 $define = [
-    'ADMIN_NAV_DATE_TIME_FORMAT' => '%A %d %b %Y %X (%Z)',
+    'ADMIN_NAV_DATE_TIME_FORMAT' => '%A %d %b %Y %X (%z)',
```

Comment:
I think this is the part that I worry about, just for upgraders. It's one more thing to have to touch that looks either insignificant or is really confusing. Just something they have to think about whether they agree with the change or not.
But, there are enough changes to this file in the upgrade that I guess it's negligible.